code/index.html

```diff
@@ -1,30 +1,107 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
-
-    <!-- Signup form -->
-
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
-
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Project 3</title>
+
+  <!-- Google font import -->
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat&display=swap" rel="stylesheet">
+
+  <!-- CSS Import -->
+  <link rel="stylesheet" href="./style.css">
+
+</head>
+
+<body>
+
+  <!-- Header container start -->
+  <div class="headerContainer">
+
+    <div class="navigation">
+
+      <div class="TabletDesktopMenu">
+        <a href="./index.html" class="TabletDesktopMenuHomeLink">Home</a>
+        <a href="./signup.html">Sign up</a>
+      </div>
+      <!-- Mobile Hamburger menu start -->
+      <div>
+        <button class="ham"></button>
+        <nav class="navbar">
+          <ul>
+            <li><a href="./index.html">Home</a></li>
+            <li><a href="./signup.html">Sign up</a></li>
+          </ul>
+        </nav>
+      </div>
+      <!-- Mobile hamburger manu end -->
+
+      <!-- Header title start -->
+      <div>
+        <h2>What the stock</h2>
+      </div>
+      <!-- Header title end -->
+
+    </div>
+
+    <!-- Hero image and text start -->
+    <div class="heroContainer">
+      <!-- Hero Image -->
+      <img src="./images/heroImage.jpg" alt="" class="headerImage">
+      <div class="heroTitle">
+        <!-- Header Title -->
+        <h1>Stock Market FAQ's</h1>
+
       </div>
+    </div>
+    <!-- Hero image and text end -->
 
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+  </div>
+  <!-- Header container end -->
+
+
+  <!-- Accordion start -->
+  <div class="accordionWrapper">
```

Comment:
Try to see if you can use some other element than `div`, it's easy to get a looot of `div`s. Maybe look into `main` and `section` for a more semantic approach?